webpack.config.js

```diff
@@ -157,20 +153,20 @@ webpackConfig.module.loaders = [{
 // Local modules can still be enabled (ie for offline development)
 // in TEST we need local modules because karma uses a different index.html (no CDNs)
 if (__TEST__ || argv.localModules) {
```

Comment:
This is more complication that I've never made use of in 2 years (maybe once?).  Unless you use it, I say we drop it.  It allows for running the project without an internet connection, provided you've already installed node_modules.  You'll still get all the network errors due to CDNs failing to load, but, the local node_modules versions will be used instead.

However, this one feature alone requires us to have the style-loader, css-loader, sass-loader, file-loader, and yargs.  All these can go away if we drop this feature.